cupy/cuda/__init__.py

```diff
@@ -81,6 +88,20 @@ def get_cuda_path():
     return _cuda_path
 
 
+def use_cub():
```

Comment:
Why do you use function call?
This is very small function. You can use single variable, for example `cub_use`. 
Or, you can avoid importing `cub if `CUB_DISABLED` is setting.

```
cub_enabled = False
if int(os.getenv('CUB_DISABLED', 0)):
    try:
        from cupy.cuda import cub  # NOQA
        cub_enabled = True
    except ImportError:
        pass
```

Reply:
Thanks for your comment. Looks it is better. I will fix it in that way.